.babelrc

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
- [ ] Please remember that one of the requirements for the video is to highlight a piece of the code that you find interesting or are proud of making, kindly record the video again and add this 😄